schema/telefonistka.json

```
@@ -0,0 +1,146 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://json.schemastore.org/telefonistka.json",
+  "title": "Schema for telefonistka.yaml files",
```

```suggestion
  "title": "Schema for repo-level telefonistka.yaml files",
```

We have two kinds of telefonistka.yaml files, repo-level that is placed in the repo root directory and [a component-level  one,](https://github.com/commercetools/telefonistka/blob/main/docs/installation.md#component-configuration) and that one has a different scheme.

We don't need to include the component-level scheme in this PR but we should name it in a way that support a later addition

---

oh, maybe also change the id field

---

I can add the component-level elements in the schema, but the completion might be confusing because it will show all available elements.

---

I thought we'll add a seperate scheme in a separate file
and the `# yaml-language-server: $schema=` URL will point to different file for each type.
But I've never used a json scheme so I'm not sure exactly how it works

---

If you're using the YAML plugin for VS Code it will automatically pull the matching schema for the file from http://www.schemastore.org/json/. We might want to make a PR to https://github.com/SchemaStore/schemastore later.

For now, we can use `# yaml-language-server: $schema` URL pointing to the raw schema JSON.